test/controllers/update/update.review.beneficial.owner.other.spec.ts

```diff
@@ -77,6 +86,52 @@ describe(`Update review beneficial owner other`, () => {
       expect(resp.text).toContain("principal addressLine1");
     });
 
+    test(`render the ${UPDATE_REVIEW_BENEFICIAL_OWNER_OTHER_PAGE} page must not contain news nocs if the flag FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC is off`, async () => {
```

Comment:
Descriptions typo - update 'news' to 'new'

Reply:
updated